test/functional/wallet_showconflictedinactivetransactions.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that listtransactions shows whether a transaction is in the mempool."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class WalletShowConflictedInactiveTransactionsTest(BitcoinTestFramework):
```

Comment:
Why not add this test in the existing `test/functional/wallet_listtransactions.py` / `ListTransactionsTest` instead of creating a new file.